unstract/workflow-execution/src/unstract/workflow_execution/execution_file_handler.py

```diff
@@ -131,7 +131,7 @@ def add_metadata_to_volume(self, input_file_path: str, source_hash: str) -> None
         if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
             file_system = FileSystem(FileStorageType.WORKFLOW_EXECUTION)
             file_storage = file_system.get_file_storage()
-            file_storage.write(path=metadata_path, mode="w", data=json.dumps(content))
+            file_storage.json_dump(path=metadata_path, mode="w", data=content)
```

Comment:
With rc2, this signature has changed. DO you want to point to rc2 and pick up the latest?

Reply:
I updated it with latest rc2, Let me check

Reply:
Removed unwanted args from  file_storage.json_dump